cv_bridge/CMakeLists.txt

```diff
@@ -90,3 +92,7 @@ install(TARGETS ${PROJECT_NAME}
 ament_package(
   CONFIG_EXTRAS "cmake/cv_bridge-extras.cmake.in"
 )
+
```

Comment:
Please move this to line 41 in src/CMakeLists.txt.

Reply:
In my tests I can only move after line 64. That works?